signer/threshold_validator.go

```diff
@@ -188,6 +192,11 @@ func (pv *ThresholdValidator) SaveLastSignedStateInitiated(chainID string, block
 			"latest_step", latest.Step,
 		)
 	}
+
+	return nil, existingTimestamp, fmt.Errorf(
```

Comment:
It doesn't look like there is a test covering this error case, and it seems important to ensure we properly handle the timeout case.